build/generate-manifests.sh

```diff
@@ -26,6 +26,16 @@ if [[ ${COMMUNITY} == "true" ]]; then
   OLM_FEATURES="[]"
   ARCHITECTURE_LABELS=$(generateArchitectureLabels amd64)
   OLM_SUBSCRIPTION_ANNOTATION=""
+  OLM_FEATURE_DISCONNECTED='"false"'
```

Comment:
Any special reason why this is quoted twice (i.e. why is the double quote included here)?

This is inconsistent with `OLM_FEATURES` (see my next comment).